.github/workflows/staticbuild.yml

```diff
@@ -176,6 +187,10 @@ jobs:
       run: |
         gsutil mv "gs://packages.viam.com/apps/viam-server/testing/static/${{ needs.static_test.outputs.date }}/${{ github.sha }}/*" "gs://packages.viam.com/apps/viam-server/"
 
+    - name: Publish Subsystem Metadata
+      run: |
+        gsutil mv "gs://packages.viam.com/apps/viam-subsystems/testing/viam-server/${{ needs.static_test.outputs.date }}/${{ github.sha }}/*" "gs://packages.viam.com/apps/viam-subsystems/"
+
```

Comment:
We do this in a separate step after-testing to avoid the risk of app picking up a bad version of viam-server that failed the test